README.md

```diff
@@ -1,35 +1,51 @@
+
+2. Install [`Docker`](https://docs.docker.com/engine/installation/), [`Singularity`](https://www.sylabs.io/guides/3.0/user-guide/) (you can follow [this tutorial](https://singularity-tutorial.github.io/01-installation/)), or [`Conda`](https://conda.io/miniconda.html). You can use conda to install Nextflow itself but use it for managing software within pipelines as a last resort. We recommend using `Docker` if possible as this has been tested most frequently. See the nf-core [docs](https://nf-co.re/usage/configuration#basic-configuration-profiles)) for more information.
+
+3. Test the pipeline on a minimal test dataset with a single command:
+
+   ```console
+   nextflow run Arcadia-Science/hifi2genome -profile test,YOURPROFILE --outdir <OUTDIR>
+   ```
 
-## Full documentation
+   Note that some form of configuration will be needed so that Nextflow knows how to fetch the required software. This is usually done in the form of a config profile (`YOURPROFILE` in the example command above). You can chain multiple config profiles in a comma-separated string.
 
-<!-- TODO: Fill in this section with how to fully use the pipeline, what the inputs are and what the outputs look like. If this section ends up being super long, feel free to create a new docs/ directory and add details there. -->
+   > - The pipeline comes with several config profiles, but we recommend using `docker` when possible, such as `-profile test,docker`.
+   > - Please check [nf-core/configs](https://github.com/nf-core/configs#documentation) to see if a custom config file to run nf-core pipelines already exists for your Institute. If so, you can simply use `-profile <institute>` in your command. This will enable either `docker` or `singularity` and set the appropriate execution settings for your local compute environment.
+   > - If you are using `singularity`, please use the [`nf-core download`](https://nf-co.re/tools/#downloading-pipelines-for-offline-use) command to download images first, before running the pipeline. Setting the [`NXF_SINGULARITY_CACHEDIR` or `singularity.cacheDir`](https://www.nextflow.io/docs/latest/singularity.html?#singularity-docker-hub) Nextflow options enables you to store and re-use the images from a central location for future pipeline runs.
+   > - If you are using `conda`, it is highly recommended to use the [`NXF_CONDA_CACHEDIR` or `conda.cacheDir`](https://www.nextflow.io/docs/latest/conda.html) settings to store the environments in a central location for future pipeline runs.
 
-## Contributions and support
+4. Start running your own analysis! The workflow expects a CSV formatted samplesheet with two columns: `sample` and `fastq` where the `sample` column contains the name of your sample (with no spaces!) and the `fastq` column contains the full path of the location of your reads in `.fastq` format. The fastq files can be in `.fq`, `.fastq` or compressed `.fq.gz` and `.fastq.gz`.
```

Comment:
Why no spaces? CSVs support spaces, is it bc of the way pipeline processes these columns?

Reply:
yeah you definitely don't want spaces in your fastq file name, the sample name unsure